orchagent/bfdorch.cpp

```diff
@@ -0,0 +1,441 @@
+#include "bfdorch.h"
+#include "intfsorch.h"
+#include "vrforch.h"
+#include "converter.h"
+#include "swssnet.h"
+#include "notifier.h"
+#include "sai_serialize.h"
+#include "directory.h"
+
+using namespace std;
+using namespace swss;
+
+#define BFD_SESSION_DEFAULT_TX_INTERVAL 1000
+#define BFD_SESSION_DEFAULT_RX_INTERVAL 1000
+#define BFD_SESSION_DEFAULT_DETECT_MULTIPLIER 3
+#define BFD_SESSION_MILLISECOND_TO_MICROSECOND 1000
+#define BFD_SRCPORTINIT 49152
+#define BFD_SRCPORTMAX 65536
+
+extern sai_bfd_api_t*       sai_bfd_api;
+extern sai_object_id_t      gSwitchId;
+extern sai_object_id_t      gVirtualRouterId;
+extern PortsOrch*           gPortsOrch;
+extern Directory<Orch*>     gDirectory;
+
+const map<string, sai_bfd_session_type_t> session_type_map =
+{
+    {"demand_active",       SAI_BFD_SESSION_TYPE_DEMAND_ACTIVE},
+    {"demand_passive",      SAI_BFD_SESSION_TYPE_DEMAND_PASSIVE},
+    {"async_active",        SAI_BFD_SESSION_TYPE_ASYNC_ACTIVE},
+    {"async_passive",       SAI_BFD_SESSION_TYPE_ASYNC_PASSIVE}
+};
+
+const map<sai_bfd_session_type_t, string> session_type_loopup =
```

Comment:
lookup -> typo?. I would suggest to name as `session_type_map`. Similarly for `session_state`

Reply:
It is.. Thanks for catching it.